SETUP.md

```diff
@@ -0,0 +1,54 @@
+# Setup
+
+## Requirements
+
+- Docker
+- Golang 1.19.2+
+
+## Prepare the working directories
+
+The directories are expected to be located in the same folder, as they depend on
+each other. These environment variables are provided to simplify the setup
+process. If you do not want to use them as is, you will need to adjust them
+according to your case.
+
+```bash
+# Export the variable(s).
+export GUACSEC_HOME="${HOME}/projects/guacsec"
+
+# Create the folders.
+mkdir -p ${GUACSEC_HOME}
+cd ${GUACSEC_HOME}
+
+# Clone the Guacsec repositories.
+git clone git@github.com:guacsec/guac-data.git
+git clone git@github.com:guacsec/guac.git
+```
+
+## Setup Neo4j
+
+Guac uses Neo4j as a graph database. Neo4j provides and maintain official Docker
+images.
+
+```bash
+docker run --rm \
+  -p7474:7474 \
+  -p7687:7687 \
+  -e NEO4J_AUTH=neo4j/s3cr3t \
+  neo4j:4.4.9-community
+```
+
+## Run Guacone
+
+Compile the binaries:
+
+```bash
+cd guac
+make build
+```
+
+and use the `guacone` client:
+
+```bash
+bin/guacone files --creds neo4j:s3cr3t ${GUACSEC_HOME}/guac-data/top-dh-sboms
+```
```

Comment:
minor: hmm, should we add a step here to tell people that they can browse the neo4j graph at `http://localhost:7474/`?

Reply:
As well as being able to use any other neo4j client like the desktop

Reply:
Addressed ✅